Playbooks/playbook-Account_Enrichment_-_Generic.yml

```diff
@@ -0,0 +1,221 @@
+id: 51af557d-bd76-483c-81b8-658c4a1e09e9
+version: -1
+name: Account Enrichment - Generic
+fromversion: 3.5.0
+description: Enrich Accounts using one or more integrations
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 03bb7183-afc8-4bcd-8f32-5c35d41fdcad
+    type: start
+    task:
+      id: 03bb7183-afc8-4bcd-8f32-5c35d41fdcad
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+  "1":
+    id: "1"
+    taskid: 74203dd2-67aa-449e-8d3b-0a37453a0426
+    type: condition
+    task:
+      id: 74203dd2-67aa-449e-8d3b-0a37453a0426
+      version: -1
+      name: Is there a username to query?
+      description: Checks if there's a username to query
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "3"
+      "yes":
+      - "2"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: general.isExists
+          left:
+            value: inputs.Username
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 193
+        }
+      }
+  "2":
+    id: "2"
+    taskid: ecd2967b-740c-4806-8a1e-77ff1b093991
+    type: title
+    task:
+      id: ecd2967b-740c-4806-8a1e-77ff1b093991
+      version: -1
+      name: Active Directory
```

Comment:
missing description, would not pass test. talk to @asafshen for script that can auto fill those